src/components/faq.js

```diff
@@ -4,10 +4,10 @@ import ReactMarkdown from "react-markdown"
 import Icon from "./Icon"
 
 const Faq = ({ data }) => {
-  // Check if data is an array before mapping
-  if (!Array.isArray(data)) {
+  // Check if data is null
+  if (!(data)) {
```

Comment:
null case should be checked with (data === null) 
is there going to be a null case or length should be checked?
please update and test. @fditata you can use ChatGPT (for future tasks) to see how this conditions are checked (when data has length or not, if is null or undefined, etc)

Reply:
@nonahayrapetyan Ok, thanks. I added an additional check for the length of data (data.length === 0).